src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -565,10 +566,9 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Specifies the type of virtual machine volumes on which to perform the encryption operation.
-Allowed values for Windows virtual machines: All, OS, and Data.
-The only allowed value for Linux virtual machines: Data.
-The VolumeType parameter is required when encrypting Linux virtual machines, but can be omitted when encrypting Windows virtual machines.
+Specifies the type of virtual machine volumes on which to perform encryption operation: OS, Data, or All. If this parameter is omitted, the operation defaults to All.
```

Comment:
Default for Linux is Data, isn't it?

Reply:
Linux currently defaults to All

Reply:
In the All case, OS + Data will be encrypted if the prerequisite setup steps for data disk encryption have been performed.  If the prerequisites have not been met, data disks, even if present, will not be encrypted.